src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -880,12 +880,18 @@ public override void VisitCoalesceExpression(ICoalesceExpression operation)
         public override void VisitIsTypeExpression(IIsTypeExpression operation)
         {
             LogString(nameof(IIsTypeExpression));
+            if (operation.IsNotTypeExpression)
+            {
+                LogString(" (IsNotExpression)");
+            }
+
             LogCommonPropertiesAndNewLine(operation);
 
             Visit(operation.Operand, "Operand");
 
             Indent();
-            LogType(operation.Type);
```

Comment:
>LogType(operation.Type); [](start = 12, length = 24)

It feels like we should keep Type property in the output.

Reply:
`LogCommonPropertiesAndNewLine` already logs the type.